field_audit/models.py

```diff
@@ -611,13 +611,23 @@ class AuditingQuerySet(models.QuerySet):
     """
 
     @validate_audit_action
-    def bulk_create(self, *args, audit_action=AuditAction.RAISE, **kw):
-        if audit_action is AuditAction.IGNORE:
-            return super().bulk_create(*args, **kw)
-        else:
-            raise NotImplementedError(
-                "Change auditing is not implemented for bulk_create()."
-            )
+    def bulk_create(self, objs, audit_action=AuditAction.RAISE, **kw):
```

Comment:
The previous `*args` was keeping this keyword-only.

```suggestion
    def bulk_create(self, objs, *, audit_action=AuditAction.RAISE, **kw):
```

Reply:
resolved in e7cad9fa596e1e853ea025be5975e8c4f977083d